# Find h5pcc command instead of h5cc.
# November 05th 2018 by B. Hajgato (Free University Brussels - VUB)
--- xmds-2.2.3/xpdeint/support/wscript.orig	2015-04-11 06:05:57.000000000 +0200
+++ xmds-2.2.3/xpdeint/support/wscript	2018-11-05 16:07:08.684961483 +0100
@@ -364,7 +364,7 @@
         )
         
         # Check for hdf5
-        if conf.find_program('h5cc', var='H5CC', mandatory = False):
+        if conf.find_program('h5pcc', var='H5CC', mandatory = False):
             h5cc_shared_flags = conf.cmd_and_log([conf.env['H5CC'], '-show', '-shlib'])
             h5cc_static_flags = conf.cmd_and_log([conf.env['H5CC'], '-show', '-noshlib'])
             
